src/ripple/ledger/View.h

```diff
@@ -340,8 +340,8 @@ transferXRP (ApplyView& view,
             STAmount const& amount,
                 beast::Journal j);
 
-NetClock::time_point const& flowV2SoTime ();
-bool flowV2Switchover (NetClock::time_point const closeTime);
+NetClock::time_point const& numAccSoTime ();
```

Comment:
naming a switchover that turns on a feature is easy. Naming switchovers that fix bugs is harder. In this case, the switchover improves numerical accuracy in some places, so I chose `numAccSwitchover`. I'll try to come up with a better name.